Ecommerce/index.css

```diff
@@ -1 +1,94 @@
-/* Write code below for ECOMMERCE TASK */
+/* Write code below for ECOMMERCE TASK */
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Welcome to My Shop</title>
+    <style>
+        body {
+            font-family: Arial, sans-serif;
+            margin: 0;
+            padding: 0;
+            background-color: #f5f5f5;
+            color: #333;
+        }
+        header {
+            background-color: #333;
+            color: #fff;
+            padding: 10px 20px;
+            text-align: center;

```

Comment:
You have added 'position: relative;' to the header. It's good practice to group related CSS properties together. Consider moving this declaration with other positioning properties like 'top', 'right', etc.